nemo_text_processing/text_normalization/en/data/address/states.tsv

```diff
@@ -28,8 +29,12 @@ Nebraska	NE
 Nevada	NV
 New Hampshire	NH
 New Jersey	NJ
+New Jersey	N.J
+New Jersey	N. J
```

Comment:
maybe make a general rule so various formats are. supported for other states as well?